packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 4.1.1
+* Add Java method @NonNull annotation
```

Comment:
A few nits here:
- This needs to follow [our style guide](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style) as noted in the checklist, so this should be "Adds", and should have a period.
- We generally put a blank line between the heading and the list, as in the rest of the file.
- Please also enclose code in `` ` ``s.
- The convention for Pigeon changes is to start with `[language]`, as you'll see in other entries.

I would suggest:
```
* [java] Adds missing `@NonNull` annotations to some methods.
```

Reply:
@stuartmorgan done